examples/reference/custom_components/AnyWidget.md

```diff
@@ -37,7 +37,7 @@ CounterButton().servable()
 :::{note}
 Panel's `AnyWidgetComponent` supports using the [`AnyWidget`](https://anywidget.dev/) API on the JavaScript side and the [`param`](https://param.holoviz.org/) parameters API on the Python side.
 
-If you are looking to create custom components using Python and Panel component only, check out [`Viewer`](Viewer.md).
+If you are looking to create custom components using Python and Panel component only, check out [`Viewer`](Viewer.md) and [`CompositeWidget`](CompositeWidget.md).
```

Comment:
As a new Panel user (and experienced) one, I think this should mention how one should go about choosing between Viewer and CompositeWidget, i.e. what scenarios should I use one over the other?

Reply:
The CompositeWidget is for widgets. The Viewer for everything Else including Panes and layouts.

Reply:
Should probably mention this since the names aren't intuitive.

Reply:
I tried to make that clear. But as you point out its to implicit and should be made more clear.